src/Controllers/ContentController.php

```diff
@@ -2225,4 +2258,122 @@ public function report(Request $request)
         return $this->sendSecure($input);
     }
 
+    public function genreCollectionPage(Request $request)
+    {
+        $types = $request->get('included_types');
+        $genre = $request->get('style');
```

Comment:
Should we do any sort of validation/safety check here, in case the `style` isn't set?

Reply:
Added. Thanks!